### ISSUE TITLE
the debian package is still broken

### ISSUE THREAD

**Issue by abeluck** [timberio/vector#6365]
### Vector Version

011.1

### Vector Configuration File
n/a

### Debug Output
n/a

### Expected Behavior

installing the deb package again should be idempotent

### Actual Behavior

The current (as of 2021-02-05)  deb package is still affected by this bug.

```
wget  https://repositories.timber.io/public/vector/deb/debian/pool/any-version/main/v/ve/vector-0.11.1-amd64.deb
````

### Additional Context

This was fixed in https://github.com/timberio/vector/commit/7ba7d772e8f3ca4e03a251a2a037a447748be966 but seems to have not made it into a new package version.

**Comment by jamtur01** [comment]
@abeluck the 0.11.1 release was prior to this fix.

**Comment by jamtur01** [comment]
Try the nightly package and you should see it is fixed. The fix will be in the forthcoming 0.12 release.
